src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -205,34 +207,43 @@
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
         if (!skip) {
-            log = new AnsiLogger(getLog(), useColor, verbose, !settings.getInteractiveMode(), getLogPrefix());
-            authConfigFactory.setLog(log);
-            imageConfigResolver.setLog(log);
+            boolean ansiRestore = Ansi.isEnabled();
+            log = new AnsiLogger(getLog(),
+                    useColor && MessageUtils.isColorEnabled()
```

Comment:
Could we please move this into an extra method, like 'useColorForLogging()' or so ....

Reply:
Sure!